crates/rpc/rpc-types/src/eth/block.rs

```diff
@@ -172,7 +172,7 @@ impl Block {
             transactions,
             total_difficulty: Some(total_difficulty),
             size: Some(U256::from(block_length)),
-            withdrawals: block.withdrawals.unwrap_or_default(),
+            withdrawals: Some(block.withdrawals.unwrap_or_default()),
```

Comment:
this will always be `Some`, this needs to check if there's a withdrawals_root in the header